source/production/F0.Analyzers/F0.Analyzers.csproj

```diff
@@ -17,10 +17,14 @@
     <PackageId>F0-Analyzers</PackageId>
   </PropertyGroup>
 
+  <PropertyGroup Label="Microsoft.CodeAnalysis.Analyzers">
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
+  </PropertyGroup>
+
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.4-beta1.22362.3" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.4" PrivateAssets="all" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.0.1" PrivateAssets="all" />
```

Comment:
Keeping the version of the _Roslyn_ packages, because updating these would be a breaking change, requiring a greater version of the .NET SDK:
See https://github.com/dotnet/roslyn/blob/main/docs/wiki/NuGet-packages.md